### DISCUSSION TITLE
HTML Token includes markdown element on the next line

### DISCUSSION THREAD

**Discussion by devintuffy** [markedjs/marked#3449] [Q&A] [answered]
Given markdown that looks like this:

```
Hello

<br>
[google.com](http://google.com)
```

Markedjs renders this HTML:

```
<p>Hello</p>
<br>
[google.com](http://google.com)
```

Note that the link has not been converted at all. This is the same for any tag on the line after an html tag:

```
Hello

<br>
*some italics*
```

becomes

```
<p>Hello</p>
<br>
*some italics
```

If I put a breakpoint in the html tokenizer, I can see that the HTML token includes the tag that follows:

```
{
    "type": "html",
    "block": true,
    "raw": "<br>\n[google.com](http://google.com)",
    "pre": false,
    "text": "<br>\n[google.com](http://google.com)"
}
```

If I remove the newline after the br tag, it renders as expected.

```
<p>Hello</p>
<p><br><a target="_blank" href="http://google.com">google.com</a></p>
```

Is this a bug? I'd expect the markdown tags to work in this case.

You can test it out on this jsfiddle:  https://jsfiddle.net/r6e25pdj/16/

**Comment by UziTech** [comment, marked as answer]
This is expected since the [commonmark spec](https://spec.commonmark.org/0.31.2/#example-155) says html blocks are ended with a blank line.

**Comment by devintuffy** [comment]
Thank you UziTech for the quick response!
